src/UserSpaceInstrumentation/TestUtils.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_DUMP_DISASSEMBLY_H_
+#define USER_SPACE_INSTRUMENTATION_DUMP_DISASSEMBLY_H_
+
+#include <cstdint>
+#include <string_view>
+#include <vector>
+
+#include "AddressRange.h"
+
+namespace orbit_user_space_instrumentation {
+
+// Returns the file offset of function `function_name` in the test executable.
+AddressRange GetFunctionAddressRangeInFileOrDie(std::string_view function_name);
+
+// Returns the virtual memory address range of function `function_name` in the test executable.
```

Comment:
Does this return the relative address in the module or the absolute address? I think this should also be clarified. From the name of the function I have contrasting signals: `GetFunctionAddress` -> relative address; `InMemory` -> absolute address (although not necessarily).

From the implementation it looks like it's the absolute address. So this can be called
`GetFunctionAbsoluteAddressRangeOrDie`.